### ISSUE TITLE
wheelmap maintenance page

### ISSUE THREAD

**Issue by holgerd** [sozialhelden/wheelmap#523]
When wheelmap is not available, I want to see a page which tells me so and shows Wheelmap's Twitter feed so I can read the latest update on why it is down and when it might be up again.

**Comment by holgerd** [comment]
Suggested wording: "Wheelmap has technical difficulties at the moment, the problem has been logged. Please check twitter for latest updates."

Then show the twitter user timeline of https://twitter.com/wheelmap

https://dev.twitter.com/web/embedded-timelines/user

**Comment by 1000miles** [comment]
Along with this ticket we'd love to also add a page for `404 - not found` as suggested here: https://github.com/sozialhelden/wheelmap/issues/137

**Comment by Hoverbear** [comment]
I'll be working on setting up a simple, standalone maintenance page that we can host on Github Pages and redirect users to when wheelmap is down and/or inaccessible.

**Comment by Hoverbear** [comment]
Can we please confirm that this will be English / German only? Here is my proposed text for English:

---

### Wheelmap is Under Maintenance

Looks like you have visited us at a bad time! We are really sorry about this. Rest assured we're working very hard to get things back in tip-top shape for you.

If you need some information about accessibility right away please head over to [OpenStreetMap](https://openstreetmap.org/). We share all of our data with the project so you will find everything you are looking for, just in a different interface!

Hope to see you later today or tomorrow when things are fixed! You can follow our latest updates on the Twitter stream below. In the meantime, keep exploring!

(Twitter feed here)

**Comment by Hoverbear** [comment]
@holgerd What are your thoughts on this first draft? https://sozialhelden.github.io/wheelmap-maintenance/

**Comment by holgerd** [comment]
@Hoverbear can you please take a minute to think about the http status code for this?

**Comment by Xylakant** [comment]
the proper status code für maintenance ist "503 Service Temporarily unavailable", eventuell gepaart mit einem Retry-After Header.

**Comment by Hoverbear** [comment]
So it seems like Github pages does not allow for returning a custom status code.

**Comment by Hoverbear** [comment]
So it seems like the best way to do this on the new infrastructure would be to have the `loadbalancer` host a small web server that holds the webpage and returns a `503`. On the existing infrastructure I'm not really sure the best way to set this up while still sending the proper status code.

**Comment by Xylakant** [comment]
I thought fabio could return the required content. However, all of that doesn't work if the LB itself needs to be taken down - in that case we'd have to redirect traffic earlier.

**Comment by Hoverbear** [comment]
Yeah, so that's part of the question... Were do we host this maintenance page?

**Comment by Xylakant** [comment]
the blog/wordpress would be a more or less obvious location.

**Comment by Hoverbear** [comment]
I agree, on the wordpress site would probably be best.

Currently on production there is this rule in the nginx file:

```
  error_page 503 @maintenance;
  error_page 500 502 504 /500.html;

  location @maintenance {
    root /var/apps/wheelmap/production/current/public;
    rewrite  ^(.*)$  /system/maintenance.html last;
    break;
  }
```

However this doesn't really help us in the case that the server is down.

**Comment by Hoverbear** [comment]
I've sent a message to @thermann78 about hosting this on the Wheelmap Blog.

**Comment by Hoverbear** [comment]
@thermann78 posted http://news.wheelmap.org/503/ today. We still need to figure out how to get it to return a 503. :)

**Comment by holgerd** [comment]
We discussed that this should be done on the new infrastructure.

**Comment by Hoverbear** [comment]
Seems to be working: https://sozialhelden.slack.com/archives/wheelmap/p1487695023000134
